docs/documentation/stories/disk-serve.md

```diff
@@ -0,0 +1,23 @@
+# Serve from Disk
+
+The CLI supports running a live browser reload experience to users by running `ng serve`. This will compile the application upon file saves and reload the browser with the newly compiled application. This is done by hosting the application in memory and serving it via [webpack-dev-server](https://webpack.js.org/guides/development/#webpack-dev-server).
+
+If you wish to get a similar experience with the application output to disk please use the steps below. This practice will allow you to ensure that serving the contents of your `dist` dir will be closer to how your application will behave when it is deployed.
+
+## Environment Setup
+### Install a web server
+You will not be using webpack-dev-server, so you will need to install a web server for the browser to request the application. There are many to choose from but a good one to try is [lite-server](https://github.com/johnpapa/lite-server) as it will auto-reload your browser when new files are output.
```

Comment:
I think a user wanting to do this would understand that its not part of the CLI. The furthest I would go is `npm - lite-server --save-dev` and add an npm script *or* `npm i lite-server -g`

because if they install it locally, they need the script.

or we skip it all. im fine with that too :)